roles/delius-core/tasks/main.yml

```diff
@@ -69,14 +69,23 @@
   become: yes
   become_user: oracle
 
-- name: (main) Cron job for nightly service restart
+- name: (main) Cron job for nightly WebLogic service restart (Every day at 2am)
   become: yes
   cron:
-    name: Restart WebLogic
-    job: 'systemctl restart weblogic'
+    name: Restart WebLogic nightly
+    job: 'timeout 5 bash -c "</dev/tcp/$(hostname)/61617" || systemctl restart weblogic'
     minute: '0'
```

Comment:
Can we push this to 15 mins past .  So we don't clash with ALF messages

Reply:
We could push it to 3am to be safe?

Reply:
Updated weekly to 00:15 and nightly to 02:15